### PR TITLE
[15.0][MIG] hr_timesheet_activity_begin_end

### PR DESCRIPTION
**Pull request by CRogos**
contains changes of #507 and depends on hr_timesheet_sheet

### CODE REVIEW

**Comment by CRogos** [comment]
@schhatbar-initos @bosd could you have a look at this? You were involved in the v14 migration.

**Comment by manuelcalerosolis** [comment]
Trying to download the module gives me this error.

Auto-merging setup/_metapackage/setup.py507)
**CONFLICT (content): Merge conflict in setup/_metapackage/setup.py**
Automatic merge failed; fix conflicts and then commit the result.

setup/_metapackage/setup.py

    install_requires=[
        'odoo-addon-hr_timesheet_task_domain>=15.0dev,<15.1dev',
        'odoo-addon-hr_timesheet_task_required>=15.0dev,<15.1dev',
        'odoo14-addon-hr_timesheet_activity_begin_end',
        'odoo14-addon-hr_timesheet_begin_end',
        'odoo14-addon-hr_timesheet_analysis',
        'odoo14-addon-hr_timesheet_sheet_begin_end',
    ],
    classifiers=[
        'Programming Language :: Python',

**Comment by CRogos** [comment]
I have rebased the branch, please try again.

**Comment by manuelcalerosolis** [comment]
Now is ok!

Thanks

**Review by manuelcalerosolis — APPROVED** [review]

**Comment by CRogos** [comment]
I've rebased and implemented the split/rename of hr_timesheet_activity_begin_end in hr_timesheet_begin_end and hr_timesheet_sheet_begin_end.

```mermaid
  graph TD;
subgraph Active
      A["hr_timesheet"]
      
      A-->B["hr_timesheet_begin_end"];
      A-->C["hr_timesheet_sheet"];
      C-->D["hr_timesheet_sheet_begin_end"];
      B-->D;
end
      subgraph Depricated
          D-->E["hr_timesheet_activity_begin_end"];
      end
```

**Comment by CRogos** [comment]
@alexey-pelykh @pedrobaeza could you have a look on this migration. It is open for quite a while and the migration is finished from my perspective.

**Review by nimarosa — APPROVED** [review]
Code review looks okay.

File: setup/hr_timesheet_begin_end/odoo/addons/hr_timesheet_begin_end [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../hr_timesheet_begin_end
```
Comment (sbidoul) [inline setup/hr_timesheet_begin_end/odoo/addons/hr_timesheet_begin_end:1]: This should be a symbolic link, other the merge will fail.

File: setup/hr_timesheet_sheet_begin_end/odoo/addons/hr_timesheet_sheet_begin_end [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../../hr_timesheet_sheet_begin_end
```
Comment (sbidoul) [inline setup/hr_timesheet_sheet_begin_end/odoo/addons/hr_timesheet_sheet_begin_end:1]: This should be a symbolic link, other the merge will fail.

**Review by sbidoul — DISMISSED** [review]
A technical remark, raised by the bot monitoring.

**Comment by nimarosa** [comment]
@CRogos please make the changes suggested by @sbidoul so we can try the merge again

**Comment by CRogos** [comment]
Ok I think I've fixed the links. Is my change causing now the build error? From the logs I assume a github connection issue?

**Review by sbidoul — COMMENTED** [review]
All green now, thanks. Please retry the merge

**Comment by EmilioPascual** [comment]
I have just added rename module to apriory in openupgrade https://github.com/OCA/OpenUpgrade/pull/4340

**Pull request merged**